common/constants.ts

```diff
@@ -38,6 +38,8 @@ export const regex = {
   username: /^[a-zA-Z0-9]{1,32}$/,
   // NOTE(jim): eight characters, at least one letter and one number.
   password: /^(?=.*[A-Za-z])(?=.*\d).{8,}$/,
+  // NOTE(stef): eight characters, at least one letter and one number.
```

Comment:
nit: This isn't necessary, you can use the same regex as the password regex

Reply:
I understand. I removed that duplicate